src/main/kotlin/day05/Collection.kt

```
@@ -0,0 +1,40 @@
+    val test = Test()
+    test.fireball { damage ->
+        println("파이어볼 발사 $damage")
+    }
+}
+
+class Man(
+    var name: String,
+    val hp: Int
+) {
+
+}
```

detekt (1.23.7)

[warning] The class or object Man is empty.

(detekt.empty-blocks.EmptyClassBlock)